### PR TITLE
Disable the logout info in the title

### LINKED ISSUE #850: Tab title is not reset after session has been locked
Steps to reproduce:
* Login to privacyIDEA, tab title is "privacyIDEA Authentication System"
* Wait until session timeout is reached
* Login dialog appears, tab title changes to "Your session has expired." and stays like that

### PR DESCRIPTION
**Pull request by cornelinux**
We totally disable the logout info in the title,
so that we do not get problems with "Your session expired",
changing and refreshing the title.

At this point also update ng-idle.

Closes #850
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/privacyidea/privacyidea/pull/856%23issuecomment-346992133%22%2C%20%22https%3A//github.com/privacyidea/privacyidea/pull/856%23issuecomment-346993209%22%2C%20%22https%3A//github.com/privacyidea/privacyidea/pull/856%23discussion_r153237338%22%2C%20%22https%3A//github.com/privacyidea/privacyidea/pull/856%23issuecomment-347445668%22%2C%20%22https%3A//github.com/privacyidea/privacyidea/pull/856%23issuecomment-347447207%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/privacyidea/privacyidea/pull/856%23issuecomment-346992133%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22A%20penny%20for%20your%20thoughts%20or%20merge%20%3A-%29%22%2C%20%22created_at%22%3A%20%222017-11-26T08%3A20%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/1908620%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cornelinux%22%7D%7D%2C%20%7B%22body%22%3A%20%22%23%20%5BCodecov%5D%28https%3A//codecov.io/gh/privacyidea/privacyidea/pull/856%3Fsrc%3Dpr%26el%3Dh1%29%20Report%5Cn%3E%20Merging%20%5B%23856%5D%28https%3A//codecov.io/gh/privacyidea/privacyidea/pull/856%3Fsrc%3Dpr%26el%3Ddesc%29%20into%20%5Bmaster%5D%28https%3A//codecov.io/gh/privacyidea/privacyidea/commit/403e86c7fcb24fff2ddabb50a10282f18a1c4e02%3Fsrc%3Dpr%26el%3Ddesc%29%20will%20%2A%2Anot%20change%2A%2A%20coverage.%5Cn%3E%20The%20diff%20coverage%20is%20%60n/a%60.%5Cn%5Cn%5B%21%5BImpacted%20file%20tree%20graph%5D%28https%3A//codecov.io/gh/privacyidea/privacyidea/pull/856/graphs/tree.svg%3Fheight%3D150%26src%3Dpr%26token%3D7nHLZki60B%26width%3D650%29%5D%28https%3A//codecov.io/gh/privacyidea/privacyidea/pull/856%3Fsrc%3Dpr%26el%3Dtree%29%5Cn%5Cn%60%60%60diff%5Cn%40%40%20%20%20%20%20%20%20%20%20%20%20Coverage%20Diff%20%20%20%20%20%20%20%20%20%20%20%40%40%5Cn%23%23%20%20%20%20%20%20%20%20%20%20%20master%20%20%20%20%20%23856%20%20%20%2B/-%20%20%20%23%23%5Cn%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%5Cn%20%20Coverage%20%20%2095.32%25%20%20%2095.32%25%20%20%20%20%20%20%20%20%20%20%20%5Cn%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%5Cn%20%20Files%20%20%20%20%20%20%20%20%20126%20%20%20%20%20%20126%20%20%20%20%20%20%20%20%20%20%20%5Cn%20%20Lines%20%20%20%20%20%20%2015565%20%20%20%2015565%20%20%20%20%20%20%20%20%20%20%20%5Cn%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%5Cn%20%20Hits%20%20%20%20%20%20%20%2014837%20%20%20%2014837%20%20%20%20%20%20%20%20%20%20%20%5Cn%20%20Misses%20%20%20%20%20%20%20%20728%20%20%20%20%20%20728%5Cn%60%60%60%5Cn%5Cn%5Cn%5Cn------%5Cn%5Cn%5BContinue%20to%20review%20full%20report%20at%20Codecov%5D%28https%3A//codecov.io/gh/privacyidea/privacyidea/pull/856%3Fsrc%3Dpr%26el%3Dcontinue%29.%5Cn%3E%20%2A%2ALegend%2A%2A%20-%20%5BClick%20here%20to%20learn%20more%5D%28https%3A//docs.codecov.io/docs/codecov-delta%29%5Cn%3E%20%60%5Cu0394%20%3D%20absolute%20%3Crelative%3E%20%28impact%29%60%2C%20%60%5Cu00f8%20%3D%20not%20affected%60%2C%20%60%3F%20%3D%20missing%20data%60%5Cn%3E%20Powered%20by%20%5BCodecov%5D%28https%3A//codecov.io/gh/privacyidea/privacyidea/pull/856%3Fsrc%3Dpr%26el%3Dfooter%29.%20Last%20update%20%5B403e86c...0dd04d4%5D%28https%3A//codecov.io/gh/privacyidea/privacyidea/pull/856%3Fsrc%3Dpr%26el%3Dlastupdated%29.%20Read%20the%20%5Bcomment%20docs%5D%28https%3A//docs.codecov.io/docs/pull-request-comments%29.%5Cn%22%2C%20%22created_at%22%3A%20%222017-11-26T08%3A44%3A57Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/8655789%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/codecov-io%22%7D%7D%2C%20%7B%22body%22%3A%20%22Looks%20good%20to%20me%21%22%2C%20%22created_at%22%3A%20%222017-11-28T08%3A13%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/28243%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/fredreichbier%22%7D%7D%2C%20%7B%22body%22%3A%20%22Merge%20it%20%3A-%29%22%2C%20%22created_at%22%3A%20%222017-11-28T08%3A20%3A15Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/1908620%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cornelinux%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%200dd04d41ae9774d0039b70710218acff9e4551ab%20privacyidea/static/contrib/js/ngmodules/angular-idle.js%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/privacyidea/privacyidea/pull/856%23discussion_r153237338%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20was%20taken%20from%20ng-idle%20upstream%20%3A-%29%5Cr%5CnI%20did%20not%20write%20this%21%20%3B-%29%20%22%2C%20%22created_at%22%3A%20%222017-11-27T15%3A55%3A31Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/1908620%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cornelinux%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20privacyidea/static/contrib/js/ngmodules/angular-idle.js%3AL1-7%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/privacyidea/privacyidea/pull/856#issuecomment-346992133'>General Comment</a></b>
- <a href='https://github.com/cornelinux'></a> A penny for your thoughts or merge :-)
- <a href='https://github.com/codecov-io'></a> # [Codecov](https://codecov.io/gh/privacyidea/privacyidea/pull/856?src=pr&el=h1) Report
> Merging [#856](https://codecov.io/gh/privacyidea/privacyidea/pull/856?src=pr&el=desc) into [master](https://codecov.io/gh/privacyidea/privacyidea/commit/403e86c7fcb24fff2ddabb50a10282f18a1c4e02?src=pr&el=desc) will **not change** coverage.
> The diff coverage is `n/a`.
[Impacted file tree graph](https://codecov.io/gh/privacyidea/privacyidea/pull/856?src=pr&el=tree)
```diff
@@           Coverage Diff           @@
##           master     #856   +/-   ##
=======================================
Coverage   95.32%   95.32%
=======================================
Files         126      126
Lines       15565    15565
=======================================
Hits        14837    14837
Misses        728      728
```
------
[Continue to review full report at Codecov](https://codecov.io/gh/privacyidea/privacyidea/pull/856?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/privacyidea/privacyidea/pull/856?src=pr&el=footer). Last update [403e86c...0dd04d4](https://codecov.io/gh/privacyidea/privacyidea/pull/856?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).
- <a href='https://github.com/fredreichbier'></a> Looks good to me!
- <a href='https://github.com/cornelinux'></a> Merge it :-)
- [ ] <a href='#crh-comment-Pull 0dd04d41ae9774d0039b70710218acff9e4551ab privacyidea/static/contrib/js/ngmodules/angular-idle.js 4'></a> &nbsp;<b><a href='https://github.com/privacyidea/privacyidea/pull/856#discussion_r153237338'>File: privacyidea/static/contrib/js/ngmodules/angular-idle.js:L1-7</a></b>
- <a href='https://github.com/cornelinux'></a> This was taken from ng-idle upstream :-)
I did not write this! ;-)

<a href='https://www.codereviewhub.com/privacyidea/privacyidea/pull/856?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/privacyidea/privacyidea/pull/856?approve=1'></a>&nbsp;<a href='https://github.com/privacyidea/privacyidea/pull/856'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by cornelinux** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/privacyidea/privacyidea/pull/856'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by cornelinux** [comment]
A penny for your thoughts or merge :-)

File: privacyidea/static/contrib/js/ngmodules/angular-idle.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*** Directives and services for responding to idle users in AngularJS
 * @author Mike Grabski <me@mikegrabski.com>
-* @version v1.1.0
+* @version v1.3.2
```
Comment (cornelinux) [inline privacyidea/static/contrib/js/ngmodules/angular-idle.js]: This was taken from ng-idle upstream :-)
I did not write this! ;-)

**Comment by fredreichbier** [comment]
Looks good to me!

**Comment by cornelinux** [comment]
Merge it :-)

**Pull request merged**